app_flair.py

```diff
@@ -71,11 +81,11 @@ def route_train_ner():
     if lock_ner_train.acquire(block=False):
         json_data = request.get_json()
         train, dev = parse_ner_train_request(json_data)
-        t1 = threading.Thread(target=train_ner, args=(train, dev))
+        t1 = threading.Thread(target=flair_train_ner, args=(train, dev))
         t1.start()
-        return ('', 204)
+        return ("", 204)
```

Comment:
What is the empty string? I assume the number is a HTTP code. The could would be more readable if constants were used instead of "magic numbers".